src/components/implementation/no_interface/vkernel/vkernel.c

```diff
@@ -111,10 +143,9 @@ cos_init(void)
 		assert(vmcc);
 
 		cos_meminfo_init(&vm_cinfo->mi, BOOT_MEM_KM_BASE, VM_UNTYPED_SIZE, vmutpt);
-		cos_compinfo_init(vm_cinfo, vmpt, vmct, vmcc,
-				(vaddr_t)BOOT_MEM_VM_BASE, VM_CAPTBL_FREE, vk_cinfo);
-		cos_compinfo_init(&vm_info->shm_cinfo, vmpt, vmct, vmcc,
-				(vaddr_t)VK_VM_SHM_BASE, VM_CAPTBL_FREE, vk_cinfo);
+		cos_compinfo_init(vm_cinfo, vmpt, vmct, vmcc, (vaddr_t)BOOT_MEM_VM_BASE, VM_CAPTBL_FREE, vk_cinfo);
+		cos_compinfo_init(
```

Comment:
This is a buggy github diff. It's fixed if you look at the actual file:
https://github.com/Others/composite/blob/ppos/src/components/implementation/no_interface/vkernel/vkernel.c#L57